src/main/java/com/shima/todo_list/controller/TodoListController.java

```diff
@@ -60,4 +62,13 @@ public ResponseEntity<UpdateResponse> updateTodoList(@PathVariable int id, @Requ
         UpdateResponse updateResponse = new UpdateResponse("Update completed!");
         return ResponseEntity.ok(updateResponse);
     }
+
+    //Delete
+    //idで指定した情報削除
+    @DeleteMapping("/todo_lists/{id}")
+    public ResponseEntity<DeleteResponse> deleteTodoList(@PathVariable int id) {
+        todoListService.delete(id);
+        DeleteResponse deleteResponse = new DeleteResponse("Information deleted");
```

Comment:
Task information なのか Information なのか揃えたほうがいいですね。
```suggestion
        DeleteResponse deleteResponse = new DeleteResponse("Task information deleted");
```

Task でもよいかと。
```suggestion
        DeleteResponse deleteResponse = new DeleteResponse("Task deleted");
```

Reply:
『Task deleted』に修正しました！

1dee368d74c3b4fbef1a8ed02f5c88e5c08ae0b7

Reply:
OKです！